pennylane/collections/map.py

```diff
@@ -104,6 +100,8 @@ def my_template(params, wires, **kwargs):
     >>> qnodes(params)
     array([-0.06154835  0.99280864])
     """
+    measure_map = {"expval": qml.expval, "var": qml.var, "sample": qml.sample}
```

Comment:
We need to move this mapping within the function, so that it is defined dynamically to take into account tape mode being turned on/off.

Reply:
This can be moved back once we are always in tape mode, right? Maybe we should have a separate list somewhere of all the things that need to be reversed and deleted...

Reply:
Yep, I'll add a todo
```suggestion
    # TODO: return measure_map to a module variable when tape-mode is default
    measure_map = {"expval": qml.expval, "var": qml.var, "sample": qml.sample}
```